locals.tf

```
@@ -20,6 +20,10 @@ locals {
       persistence = {
         defaultClass             = var.enable_pv_backups && var.set_default_storage_class ? "false" : "true"
```

```suggestion
        defaultClass             = tostring(var.enable_pv_backups && var.set_default_storage_class)
```

Nothing to do with your PR, but we could change this also like the other one you changed earlier.